### DISCUSSION TITLE
Tasmota Smart Meter Interface - is there a special branch?

### DISCUSSION THREAD

**Discussion by francwalter** [arendst/Tasmota#22336] [Q&A] [answered]
Hallo
is there a special branch for the [Smart Meter Interface of Tasmota](https://tasmota.github.io/docs/Smart-Meter-Interface/) where all the typical Tasmota features are missing, especially the Update (OTA) Button?

It is not possible to update that special compiled version, it would update to the normal Tasmota, so why not throw out all this unrelevant functions? 
Only to serve for Smart Meter purposes?

Sorry if this is a silly question, I am quite new to that 😊 
Thank frank

**Comment by Jason2866** [comment, marked as answer]
No, there is no special branch for Smart Meter SML.
